app/src/main/java/ch/epfl/sdp/appart/PanoramaActivity.java

```diff
@@ -136,14 +149,7 @@ public void goRight(View view) {
     /**
      * Init the list of panorama images, set the current image index to 0, load image.
      */
-    private void getImages() {
-        images = new ArrayList<>();
-        // TODO change with database call to get image references
-        images.add("file:///android_asset/panorama_test.jpg");
-        images.add("file:///android_asset/panorama_test_2.jpg");
-        //images.add("file:///android_asset/panorama_test_3.jpg");
-        //images.add("file:///android_asset/panorama_test_4.jpg");
-    }
+    private void getImages() {}
```

Comment:
Why this function is empty ? maybe is better remove or adding a TODO comment.

Reply:
Good catch, it should be removed.